include/deal.II/base/index_set.h

```diff
@@ -218,6 +218,15 @@ class IndexSet
   bool is_contiguous () const;
 
   /**
+   * Return whether the index set stored by this object defines a linear
+   * range, i.e., each index is contained in exactly one IndexSet,
+   * the first indices are contained in the IndexSet of the first MPI process,
+   * the second indices are contained in the IndexSet of the second MPI process
+   * and so on.
+   */
+  bool is_linear(const MPI_Comm &communicator) const;
```

Comment:
I don't think linear is the correct word here. "Distributed in ascending order" or something? What happens if you have no MPI in this function (document that)?